test/mocha.opts

```diff
@@ -0,0 +1,7 @@
+--ui bdd
+--check-leaks
+--recursive
+--slow 200
+--reporter spec
+--compilers js:babel-core/register
```

Comment:
- [ ] This should be `--compilers js:babel-register`, also run `npm i -D babel-register`